.goreleaser.yaml

```diff
@@ -2,11 +2,9 @@ builds:
   - id: ldcli
     binary: ldcli
     goos:
-      - linux
       - darwin
-    goarch:
```

Comment:
I'm not sure we need to specify the goarch values. When I generated a default goreleaser config file it only specifies the goos values.

Reply:
If you don't specify goarch, the docs say the default is `'386', 'amd64', 'arm64'`. So not sure we need to build for 32 bit processors, but it probably wouldn't hurt.

Reply:
![Screenshot 2024-06-03 at 4 12 19 PM](https://github.com/launchdarkly/ldcli/assets/306396/218cefa1-a35f-4425-b28f-ee08cd664849)
Updated with defined GOARCH values.